package.json

```diff
@@ -79,6 +80,7 @@
     "camelcase-keys-deep": "^0.1.0",
     "capacitor-native-settings": "^6.0.1",
     "cordova-plugin-screen-orientation": "^3.0.4",
+    "crypto-js": "^4.2.0",
```

Comment:
> Active development of CryptoJS has been discontinued. This library is no longer maintained.

https://www.npmjs.com/package/crypto-js

I also would check how it affects the bundle size. Consider using [Web Crypto](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) instead.